docbook/gnc-locale-C.dtd

```diff
@@ -1,8 +1,6 @@
 <!-- Add master English entities that should be translated below.
      Only parameter entities (% entities) should be translated.
-     See htps://www.w3.org/TR/REC-xml/#sec-physical-struct. -->
-
-
+     See https://www.w3.org/TR/REC-xml/#sec-physical-struct. -->
```

Comment:
should we add something like
As an example compare this file with the german localization gnc-locale-de.dtd.

Perhaps we should explain it better for "simple" translators:

Motivation:
With this magic we have only one place to maintain menu entries and it is consistent for the user in the whole document—without typos.

Task:
Start Gnucash in your language and update this list to your menu entries.

As the list is overwelming on the first view: Which part is finally used in the docs?

Reply:
> As the list is overwelming on the first view: 

This file is not really clear, maybe we can think of something else to structure it? I think if we can split the file even further, one with just the GUI entries for translation and another with the entity combinations for use, then this can become more manageable.

> Which part is finally used in the docs?

In GUIMenus.xml most of the individual parts are used. The entities for use in the respective chapters are still to be declared as menuchoice so that they can be used in the text.